src/main/java/seedu/address/model/task/Task.java

```diff
@@ -8,7 +8,7 @@
  * Represents a Task in the address book.
  * Guarantees: Is a POJO. Details are present and not null. Field values are validated and immutable.
  */
-public abstract class Task {
+public class Task {
```

Comment:
works as in: add deadline (eg. `add potato tdy 3pm`, eg. `add potato 3pm, tdy`)  and float (`add potato 2` `add potato`) works, but event wont work because EventParser does not yet recognise `to`